man/ronin-archive.1.md

```diff
@@ -0,0 +1,40 @@
+# ronin-archive 1 "2024-01-18" Ronin "User Manuals"
+
+## NAME
+
+ronin-archive - Archive the files.
+
+## SYNOPSIS
+
+`ronin archive` [*options*] {`-o`,`--output` *PATH* } [*FILE* ...]
```

Comment:
Remove the `[` `]` from `[*FILE* ...]` since at least one `*FILE*` must be specified.